test/csuite/incr_backup/main.c

```diff
@@ -79,7 +78,7 @@ static bool do_rename = true;
  */
 typedef struct {
     char *name;            /* non-null entries represent tables in use */
-    uint32_t name_index;   /* bumped when we rename or drop, so we get unique names. */
+    uint32_t name_index;   /* bumped when drop, so we get unique names. */
```

Comment:
The "we" should not have been deleted here.

Reply:
good spot [657d423](https://github.com/wiredtiger/wiredtiger/pull/10306/commits/657d42348c32dbe6428c2101c1307ea781867e2e)